### PR TITLE
Fix ADB reverse link + release build instructions

### PR DESCRIPTION
**Pull request by andrewrabon**
ADB reverse link was broken, and the "release build" instructions did not work (--configuration doesn't exist.)

### CODE REVIEW

File: docs/RunningOnDevice.md [hunk 1]
```diff
@@ -145,11 +145,11 @@ Type the following in your command prompt to install and launch your app on the
 $ react-native run-android
 ```
 
-> If you get a "bridge configuration isn't available" error, see [Using adb reverse](#using-adb-reverse).
+> If you get a "bridge configuration isn't available" error, see [Using adb reverse](docs/running-on-device.html#method-1-using-adb-reverse-recommended).
```
Comment (hramos) [inline docs/RunningOnDevice.md]: It looks like it's just the fragment that is wrong. Can this simply be `#method-1-using-adb-reverse-recommended`?

Comment (andrewrabon) [inline docs/RunningOnDevice.md]: Doing that seems to link directly to https://facebook.github.io/react-native/#method-1-using-adb-reverse-recommended (like the currently live site.)

Comment (hramos) [inline docs/RunningOnDevice.md]: Ah, interesting. Let's merge this in then.

Comment (andrewrabon) [inline docs/RunningOnDevice.md]: Awesome, thank you!

**Pull request closed without merging**